rel-eng/tito.props

```diff
@@ -459,7 +459,7 @@ whitelist = ansiblerole-foreman_scap_client
   rubygem-smart_proxy_salt
   rubygem-smart_proxy_spacewalk
 
-[foreman-client-nightly-rhel7]
+[foreman-client-1.24-rhel7]
 disttag = .el7
 whitelist =
   # don't whitelist foreman-release here, it's already in foreman-nightly-nonscl-rhel7
```

Comment:
FYI: this is a limitation in obal. This is because obal can't release the same package to multiple tags and you must manually tag it in after every release. You may want to update the comment to refer to 1.24 instead of nightly as well.